modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -11,21 +11,23 @@
     @inactive="modal.active = false" @show-modal="modal.showModal = true"
     @esc="cancel" @no-modal="$emit('safe-close')"
   >
-    <template v-if="relationshipField" #primaryControls>
+    <template v-if="relationshipField" #secondaryControls>
       <AposButton
         type="default" label="Cancel"
         @click="cancel"
       />
+    </template>
+    <template v-else #secondaryControls>
       <AposButton
-        :label="`Save`" type="primary"
-        :disabled="relationshipErrors === 'min'"
-        @click="saveRelationship"
+        type="default" label="Exit"
+        @click="cancel"
       />
     </template>
-    <template v-else #primaryControls>
+    <template v-if="relationshipField" #primaryControls>
```

Comment:
Why are relationship fields involved in the base AposMediaManager component?

Reply:
All of the managers are also used as relationship choosers. In that context they have different buttons.